lib/src/clojure_lsp/kondo.clj

```diff
@@ -135,13 +134,10 @@
     (let [filename (-> analysis :var-definitions first :filename)
           updated-analysis (assoc (:analysis @db) filename (normalize-analysis analysis))]
       (if (settings/get db [:linters :clj-kondo :async-custom-lint?] true)
-        (async/go-loop [tries 1]
-          (if (>= tries 200)
-            (logger/info "Max tries reached when async custom linting" uri)
-            (if (contains? (:processing-changes @db) uri)
-              (do
-                (Thread/sleep 50)
-                (recur (inc tries)))
+        (future
```

Comment:
why a future and not a go/block which is less expensive?

Reply:
Also, why we are removing the loop here?

Reply:
I think they're equally as expensive, but if you know differently, I'd love to learn. From what I understand `async/go` (or probably `async/thread` in this case) is necessary if you need to pass the result of the body to other `async` code, but otherwise isn't needed. Aside from that, the only real difference is that `future` and `async/thread` get dispatched on different thread pools.

Reply:
I reverted this, so best to ignore.